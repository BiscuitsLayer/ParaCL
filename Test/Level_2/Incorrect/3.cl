temp = func (x, y) : factorial {
    if (x < 1) {
        return 1;
    } 
    return factorial (x - 1, 0) * x;
}

print temp (5, 1 / 0) * factorial (5 / 0, 7 / 0);