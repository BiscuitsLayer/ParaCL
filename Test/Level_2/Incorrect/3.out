Division by zero! (runtime)
print (temp (5, (1 / 0)) {0} * factorial ((5 / 0), (7 / 0)) {0})